tests/shared/file-extensions.spec.js

```diff
@@ -29,62 +29,66 @@ describe('Shared: fileExtensions', function() {
       // No extension
       {
         file: 'path/to/filename',
-        extension: []
+        extensions: []
       },
 
       // Type: 'archive'
       {
         file: 'path/to/filename.zip',
-        extension: [ 'zip' ]
+        extensions: [ 'zip' ]
       },
       {
         file: 'path/to/filename.etch',
-        extension: [ 'etch' ]
+        extensions: [ 'etch' ]
       },
 
       // Type: 'compressed'
       {
         file: 'path/to/filename.img.gz',
-        extension: [ 'img', 'gz' ]
+        extensions: [ 'img', 'gz' ]
       },
       {
         file: 'path/to/filename.img.bz2',
-        extension: [ 'img', 'bz2' ]
+        extensions: [ 'img', 'bz2' ]
       },
       {
         file: 'path/to/filename.img.xz',
-        extension: [ 'img', 'xz' ]
+        extensions: [ 'img', 'xz' ]
+      },
+      {
+        file: 'path/to/filename.img.gz',
```

Comment:
Well as long as you're happy for me to keep catching the little slip-ups you make ;-)